syft/workers/websocket_server.py

```diff
@@ -103,6 +109,12 @@ def __init__(
             # send the response
             await websocket.send(response)
 
+    def _recv_msg(self, message: bin) -> bin:
+        try:
+            return self.recv_msg(message)
+        except ResponseSignatureError as e:
```

Comment:
Should we do this for all exceptions?

Reply:
Not sure, for debug purpose maybe, but it depends on the usecases:
*Definitely*
- Exception which are normally in use like ResponseSignatureError and are normally caught
*Surely*
- Exception related to FL stuff like GetPermittedError, WorkerNotFoundException, TensorsNotCollocatedException etc
*Maybe not*
- Exception unrelated to FL which cause a worker to die (MemoryError, etc), in which case we just want to launch a new worker